autoPyTorch/pipeline/components/training/trainer/RowCutOutTrainer.py

```diff
@@ -37,7 +37,7 @@ def data_preparation(self, X: np.ndarray, y: np.ndarray,
             return X, {'y_a': y_a, 'y_b': y_b, 'lam': lam}
 
         size = X.shape[1]
-        indices = self.random_state.choice(range(1, size), max(1, np.int32(size * self.patch_ratio)),
+        indices = self.random_state.choice(range(size), max(1, np.int32(size * self.patch_ratio)),
```

Comment:
What happens when we switch `np.int32` to `int`?
I usually get unexpected errors when mixing up the usage of `torch` and `np.intxx`, so I would like to avoid using `np.intxx` for a single variable as much as possible.

Reply:
That is a good question

Reply:
okay I'll change this to np.int.

Reply:
Not `np.int`, but `int`!
It is a shame that I cannot provide the example of the error, but I once got an error regarding `np.int32(x)` vs `int(x)`.
And because torch says `expected int, but got int` although it was `np.int32`, it was quite hard time for me to find out why this error comes up again and again.

Reply:
hmm, weird. Maybe we can look out for the error? I have never had a problem with this and we have not faced any issues so far also related to this so maybe there was some other contributing factor to the error you got.

Reply:
Yeah, it was more or less a torch issue, so I will report when I find the same issue in the future.
But, just in case, are there any reasons to use `np.int32(x)`?
```
import numpy as np


a, b = np.int32(3), int(3)
type(a) == type(b)

>>> False
```